src/wallet/rpcdump.cpp

```diff
@@ -29,6 +29,7 @@
 
 using namespace std;
 
+CWallet *GetWalletForJSONRPCRequest(const JSONRPCRequest&);
 void EnsureWalletIsUnlocked(CWallet *);
 bool EnsureWalletIsAvailable(CWallet *, bool avoidException);
```

Comment:
Can we move these to some rpc.h in src/wallet? Why are we declaring functions in another file from the definition if they're both in src/wallet???